src/gpb.erl

```diff
@@ -34,8 +34,8 @@
 -export([defs_records_to_proplists/1,  proplists_to_defs_records/1]).
 -export([rpc_records_to_proplists/1, rpc_record_to_proplist/1, proplists_to_rpc_records/1]).
 -include_lib("eunit/include/eunit.hrl").
--include("../include/gpb.hrl").
--include("../include/gpb_version.hrl").
+-include("gpb.hrl").
```

Comment:
Yea, I'm undoing this and squashing. I had done it because I thought it was causing an issue with the tests at one point but was wrong.